pyvelox/test/test_signatures.py

```diff
@@ -41,9 +41,11 @@ def test_function_signature(self):
         concat_signatures = presto_signatures["concat"]
         self.assertTrue(len(concat_signatures) > 0)
         # Array functions are registered first, then string functions.
```

Comment:
> # Array functions are registered first, then string functions.

What is this comment about? Is it accurate?

Reply:
We do register array functions first (https://github.com/facebookincubator/velox/blob/main/velox/functions/prestosql/registration/RegistrationFunctions.cpp#L91) , but there is no guarantee the order inside this. If I am not mistaken this change changed the signatures : https://github.com/facebookincubator/velox/pull/7393/files#diff-05730d551b2432f68b64db0e73bccd5846780904db3084de78461ed5b75c5791 . 
A question could be asked why we picked 'concat' , that was purely arbitary initially.